keps/0034-instance-health.md

```diff
@@ -43,3 +43,58 @@ Drift detection (detecting that resource was deleted or changed manually)
 Including other types of resources than *readiness phase 1 resources*
 Determining if the underlying application is functional
 Determining if the underlying application is reachable
+
+## Proposal
+
+Readiness of an `Instance` will be communicated via a newly introduced `Status.Conditions` field. The schema of this field will [conform the schema](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L1367) recommended by k8s api machinery. The `Type` of the newly added condition will be `Ready`. Condition is supposed to have these three values which will have the following meaning:
```

Comment:
Is it `Status.Conditions` or `Status.Condition`?

Reply:
yeah, exactly, thanks, I rephrased a bit